<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">

    <!-- For Real Hardware Interface -->
    <ros2_control name="ActBotSys" type="system">
        <hardware>
            <plugin>BotHardwareSystem</plugin>
            <param name="left_wheel_name">left_front_wheel_joint</param>
            <param name="right_wheel_name">right_front_wheel_joint</param>
            <param name="loop_rate">50</param>
            <param name="device">/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.2:1.0-port0</param>
            <!-- <param name="device">/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1.3:1.0-port0</param> -->
            <param name="baud_rate">115200</param>
            <param name="timeout_ms">0</param>
            <param name="enc_counts_per_rev">60</param>
            <param name="pid_p">5</param>
            <param name="pid_d">30</param>
            <param name="pid_i">0</param>
            <param name="pid_o">1</param>

            <param name="left_range_name">left_range</param>
            <param name="right_range_name">right_range</param>

            <param name="imu_name">imu</param>
        </hardware>
    </ros2_control>
    
</robot>